nevergrad/functions/images/core.py

```diff
@@ -49,18 +50,10 @@ def __init__(self, problem_name: str = "recovering", index: int = 0) -> None:
         array.set_recombination(ng.p.mutation.Crossover(axis=(0, 1), max_size=max_size)).set_name("")  # type: ignore
 
         super().__init__(self._loss, array)
-        self.register_initialization(problem_name=problem_name, index=index)
-        self._descriptors.update(problem_name=problem_name, index=index)
+        self.loss_function = loss(reference=self.data)
```

Comment:
just do
`super().__init__(loss(reference=self.data), array)`
Defining `_loss` does not bring anything